src/__tests__/surveys.test.ts

```diff
@@ -275,6 +278,36 @@ describe('surveys', () => {
             start_date: new Date().toISOString(),
             end_date: null,
         } as unknown as Survey
+        const surveyWithMissingInternalFlag: Survey = {
+            name: 'survey with enabled internal flags',
```

Comment:
nit: technically this isn't the correct name (should be "survey with missing internal flags".  I know it's a test, but figured I'd flag it anyway.